lib/Service/AbstractService.php

```diff
@@ -45,31 +45,61 @@ protected function allObjects($params, $opts)
         return $this->request("get", $this->basePath(), $params, $opts);
     }
 
+    protected function allNestedObjects($nestedPath, $parentId, $params, $opts)
+    {
+        return $this->request("get", $this->baseNestedPath($parentId, $nestedPath), $params, $opts);
+    }
+
```

Comment:
Minor but as much as I like alphabetization, I think I'd rather have all nested methods together. Maybe an argument to do nestedObjectsAll or similar? No strong feels though.

Reply:
Going to keep as-is for now, I think we'll probably get rid of the helper methods anyway when we codegen the services and just use `request()` directly.